### ISSUE TITLE
Metrics for icon inclusion

### ISSUE THREAD

**Issue by NovaGL** [simple-icons/simple-icons#3204]
I think as this icon list gets bigger and bigger there needs to be more scrutiny on what icons are accepted.

I personally don't believe usage is a good enough metric.

Say you have a game mod for a basketball game and its popular and each year they release a new mod with a new icon

So you'd have
- NBA18
- NBAI9
- NBA2K

The same thing with entertainment. Is it acceptable to have every TV station in the world added.

The bottom line is what is the best use can for this icon set and do icons fit that criteria otherwise it's just a free for all with no real purpose.

**Comment by micwoj92** [comment]
Maybe as addition to current metrics, there should be field called "purpose" in which the requestor can say on why they need the icon.

**Comment by NovaGL** [comment]
Possibly, we really need to get the opinions of the maintainers.
I don't want this to be i'm trying to enforce gate keeping, just a better understanding of what this repo is trying to achieve.

Maybe its just simply an brief explanation of what simple icons is in the readme.

**Comment by NovaGL** [comment]
@ericcornelissen Can you tell me how you deal with legalities of adding logos? Eg I marked some logos as pending as I don't believe we would obtain the rights to include these logos. Eg USPS.

**Comment by ericcornelissen** [comment]
As per the README, simple icons is a collection of icons of **popular** brands.

In some cases, this may lead us down a path where we are adding icons with no real purpose (e.g. adding a bunch of TV stations, though definitely not **every** TV station in the world is popular enough). I would personally assume that most people requesting an icon here do in fact have a use case for it. Hence, I would think asking people to state a "purpose" for their icon request is not really helpful and I suspect will result in mostly superficial answers. But if others think it would be helpful I'm totally okay with adding such a field to the issue template.

In addition, we, as maintainers, are not here to fulfil every icon request but simply to check for the popularity of the brands and the quality of icons. If someone really wants a particular brand to be included they will likely need to put in the work themselves (unless one of the maintainers also wants the brand to be included).

Regarding legalities of logos: if the brand says the logo cannot be used for what our icons are typically used for we cannot include it. It's unfortunate, but not our problem. If it's unclear we usually include it, but it's really a case-by-case problem.

**Comment by PeterShaggyNoble** [comment]
Some good points here, @NovaGL. The alexa rank of a site is our primary metric and cut-off; anything above 500k and it's definitely within our scope. Although, we do have precedent for rejecting requests that meet that requirement but we deem to be too niche or regional, viewing their Alexa rank as anomalous. Those _are_ few and far between, though. Outside of Alexa ranks, we do have a few other metrics we use (e.g., 5k stars for GitHub projects), which we're still working on the cut-offs for and is why they don't yet appear in our guidelines.

To address your specific examples, I'd agree that adding the logo for each annual release of a game would be overkill. Instead we should add the more generic logo for the game series, if one exists, though. Having said that, though, if someone does request an icon for a specific game in a series then we should still take it under consideration rather than rejecting it out of hand as the release of a new version doesn't immediately render the previous version obsolete. So a general rule of thumb there could be that if the game's manufacturer is still supporting it, we consider it but, if it's old enough for the manufacturer to have withdrawn support then we would be more likely to reject it. There are other things, though, that we should reject out of hand such as the logos for individual sporting events which, for the most part, change for each event and, once the event is over, they are no longer relevant. The Olympic rings, for example, would be fine (I know: bad example, we've already rejected that one for a different reason!) but the logo for Tokyo 2021 wouldn't.

TV stations (along with periodicals and transport networks) around the world I also agree we shouldn't be adding them all but most of them should be, and have been, assessed on a case-by-case basis. Taking el [Diario de Ibiza](https://www.diariodeibiza.es/) as an example it, surprisingly, has an Alexa rank of [~158](https://www.alexa.com/siteinfo/diariodeibiza.es) which should make it within our scope but, given that the island only has a population of ~148k, it suddenly becomes very niche and more likely to be rejected.

On requesting use-cases, while I'm certainly curious myself on occasion, I think, as @ericcornelissen said, the very fact that someone is requesting we add an icon is enough to demonstrate that they _do_ have a use-case for it.

**Comment by NovaGL** [comment]
Thanks guys. I think i'm going to close this issue for the moment. If we decide to flesh out the readme and have a more descriptive intro we can re-open this issue.
